From 212b70149bc943de7180aa8c18e1f92e1cd76ec7 Mon Sep 17 00:00:00 2001
From: Bert Huijben <rhuijben@apache.org>
Date: Wed, 4 Oct 2017 14:56:22 +0000
Subject: [PATCH 2/3] Fix syntax of a print() in the scons file to unbreak
 building with most recent scons version.

* SConstruct
  Use Python 3.0 valid syntax to make Scons 3.0.0 happy on both python 3.0
  and 2.7.

git-svn-id: https://svn.apache.org/repos/asf/serf/trunk@1811083 13f79535-47bb-0310-9956-ffa450edef68
---
 SConstruct | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git SConstruct SConstruct
index 1766870..a7e18da 100644
--- SConstruct
+++ SConstruct
@@ -183,7 +183,7 @@ CALLOUT_OKAY = not (env.GetOption('clean') or env.GetOption('help'))
 
 unknown = opts.UnknownVariables()
 if unknown:
-  print 'Warning: Used unknown variables:', ', '.join(unknown.keys())
+  print('Warning: Used unknown variables:', ', '.join(unknown.keys()))
 
 apr = str(env['APR'])
 apu = str(env['APU'])
-- 
2.29.0.rc1

